closed/custom/common/Modules.gmk

```diff
@@ -23,6 +23,13 @@ BOOT_MODULES += \
     openj9.sharedclasses \
     #
 
+PLATFORM_MODULES += \
+	openj9.cuda \
+	openj9.dataaccess \
+	openj9.gpu \
+	openj9.zosconditionhandling \
+	#
+
```

Comment:
Shouldn't we consider adding this?
```
ifeq ($(OPENJDK_TARGET_OS), zos)
  PLATFORM_MODULES += ibm.jzos
endif
```

Reply:
There is a more complicated rule that does this in the zos repo, where it goes through all the additional components and adds them to the PLATFORM_MODULES list.